apps/src/templates/BaseDialog.jsx

```diff
@@ -88,7 +88,9 @@ var BaseDialog = React.createClass({
       modalBodyStyle = {
         background: `#fff top center url(${this.props.assetUrl('media/dialog/achievement_background.png')}) no-repeat`,
         height: this.props.fixedHeight,
-        overflow: 'hidden',
+        maxHeight: !this.props.fixedHeight && '80vh',
+        overflowX: 'hidden',
+        overflowY: this.props.fixedHeight ? 'hidden' : 'auto',
```

Comment:
Well, I hate to add _more_ to this dialog but I'm not sure it's time to break out another component yet.  Thinking that should happen soon though.